src/app/modules/backbone/pages/backbone/backbone.page.scss

```diff
@@ -0,0 +1,14 @@
+.backbone-container {
+    @apply grid;
+    @apply w-full;
+
+    @apply md:grid-cols-12;
+}
+
+.sidebar {
+    @apply md:col-span-3 md:h-screen;
```

Comment:
the media queries on the backbone component make it easier to define a generalized grid layout that changes based on the device used.

Here, we will move the sidebar to its own 3 columns on large screen but default to regular behaviour on small screens (top to bottom)